src/utils/AutoControlledComponent-slim.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+import _ from 'lodash'
+import { Component } from 'react'
+
+const toDefault = (prop) => `default${prop[0].toUpperCase() + prop.slice(1)}`
+
+export default class AutoControlledComponent extends Component {
+  componentWillMount() {
+    if (super.componentWillMount) super.componentWillMount()
+    const { autoControlledProps } = this.constructor
+
+    this.state = _.transform(autoControlledProps, (res, prop) => {
```

Comment:
react-controllables went with `fromDefaultName`, I'm indifferent but want it to make the most sense to the most people.